.github/actions/setup-earthly-runner/action.yml

```diff
@@ -0,0 +1,36 @@
+name: "Setup earthly runner"
+description: "Set up mTLS for connection to the remote buildkit"
+author: "ahmed-nour-fdc"
+inputs:
+  remote_buildkit_host:
+    description: "Earthly remote buildkit host"
+    required: true
+  tlsca:
+    description: "Certificate for the certificate authority"
+    required: true
+  tlscert:
+    description: "TLS Certificate"
+    required: true
+  tlskey:
+    description: "TLS key"
+    required: true
+  tls_enabled:
+    description: "Enable TLS"
+    required: false
+    default: "true"
+runs:
+  using: "composite"
+  steps:
+    - shell: sh
+      run: |
+        mkdir -p ~/certs
+        chmod 700 ~/certs
+        echo "${{ inputs.tlsca }}" > ~/certs/CACert.pem
```

Comment:
do these secrets show up in logs?

Reply:
No. Github is smart about hiding secrets. [Example](https://github.com/freiheit-com/kuberpult/actions/runs/8173444121/job/22348737824#step:4:18)